src/Application/Characters/Queries/GetUserCharacterCompetitiveRatingQuery.cs

```diff
@@ -0,0 +1,40 @@
+using AutoMapper;
+using Crpg.Application.Characters.Models;
+using Crpg.Application.Common.Interfaces;
+using Crpg.Application.Common.Mediator;
+using Crpg.Application.Common.Results;
+using Crpg.Application.Common.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace Crpg.Application.Characters.Queries;
+
+public record GetUserCharacterCompetitiveRatingQuery : IMediatorRequest<CharacterCompetitiveRatingViewModel>
```

Comment:
Can be merged in the get character rating query too.

Reply:
what get character rating query?